src/SFA.DAS.AssessorService.EpaoDataSync/RefreshIlrsHttpFuncionApp.cs

```diff
@@ -11,26 +11,26 @@
 
 namespace SFA.DAS.AssessorService.EpaoDataSync
 {
-    public static class RefreshIlrsFuncionApp
+    public static class RefreshIlrsHttpFuncionApp
```

Comment:
Can you fix the typo in this class please? :)

Reply:
Corrected